website/docs/components/tag/partials/guidelines/guidelines.md

```diff
@@ -1,82 +1,55 @@
-## When to use
+## Usage
 
-- Use tags to indicate an object's categorization, i.e., for filtering.
+### When to use
 
-## When not to use
+Use tags to indicate an object's categorization, i.e., for filtering.
 
-- for static metadata, status, or to display a new feature, use [Badge](/components/badge/overview)
-- For collection enumeration or version number, use [BadgeCount](/components/badge-count/overview)
-
----
-
-## Anatomy
-
-![Tag anatomy](/assets/components/tag/tag-anatomy.png)
-
-#### Text
+### When not to use
 
-Required
-
-#### Icon
-
-Required for isDismissible
-
-#### Container
-
-Required
-
----
+- For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview)
+- For collection enumeration or version number, use [BadgeCount](/components/badge-count/overview)
 
-## Layout
+### Layout
 
 - The Tag component can be seen in several different layouts, but we commonly see them in a list or a gorup.
 - Tags will fill the parent container and then wrap, as necessary
 
-### List
+#### List
 
 ![Layout list example](/assets/components/tag/tag-layout-list.png)
 
-### Group
+#### Group
 
 ![Layout group example](/assets/components/tag/tag-layout-group.png)
 
 ---
 
-## Spacing
+### Spacing
 
 As a best practice, we recommend a vertical spacing of 12px and horiztonal spacing of 8px between tags.
```

Comment:
As a best practice, we recommend a vertical spacing of 12px and horizontal spacing of 8px between tags.